.eslintrc.json

```diff
@@ -0,0 +1,75 @@
+{
+  "env": {
+      "browser": true,
+      "es6": true
+  },
+  "extends": "airbnb",
+  "parser": "babel-eslint",
+  "globals": {
+    "window": false,
+    "navigator": false,
+    "document": false,
+    "location": false,
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+      "ecmaFeatures": {
+          "jsx": true
+      },
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+  },
+  "plugins": [
+      "react"
+  ],
+  "rules": {
+    "consistent-return": "off",
+    "no-plusplus": "off",
+    "no-mixed-operators": "off",
+    "no-bitwise": "off",
+    "no-underscore-dangle": "off",
+    "no-multiple-empty-lines": "off",
```

Comment:
I think this one should be on at its default level to align with our normal linting rules

Reply:
Done